person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] [Optional] [Highly recommended] It would be better coded if you make the `id` instance generated automatically, not passed as a parameter while creating a new person as you did. It could be something like that:
```ruby
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
end
```